docs/elements/reference/sign-in.mdx

```diff
@@ -34,6 +34,7 @@ The root sign-in component. Sets up providers and state management for the sign-
 | ---- | ------------------- | ----------------------------------------------------------------- |
 | `path?` | `string` | The root path the sign-in flow is mounted at. Default: `/sign-in` |
 | `fallback?` | `React.ReactNode` | Fallback markup to render while Clerk is loading. Default: `null` |
+| `routing?` | `'path' \| 'virtual'` | If you want to render Clerk Elements in e.g. a modal, use the `'virtual'` routing mode. Default: `'path'` |
```

Comment:
@LekoArts we don't have hash routing here, right?

Reply:
Nope, only those two atm 👍